centrifuge-app/src/components/InvestRedeem/InvestRedeemLiquidityPoolsProvider.tsx

```diff
@@ -174,20 +183,30 @@ export function InvestRedeemLiquidityPoolsProvider({ poolId, trancheId, children
       ? {
           investCurrency: lpInvest.pendingInvest.toDecimal(),
           redeemToken: lpInvest.pendingRedeem.toDecimal(),
-          payoutCurrencyAmount: lpInvest.maxDeposit.toDecimal(),
+          payoutCurrencyAmount: lpInvest.maxWithdraw.toDecimal(),
           payoutTokenAmount: lpInvest.maxMint.toDecimal(),
+          investClaimableCurrencyAmount: lpInvest.maxDeposit.toDecimal(),
+          redeemClaimableTokenAmount: lpInvest.maxWithdraw.toDecimal(),
```

Comment:
If this is supposed to be the # of tranche tokens:
```suggestion
          redeemClaimableTokenAmount: lpInvest.maxRedeem.toDecimal(),
```